lib/generators/action_view/component/templates/component.rb.tt

```diff
@@ -0,0 +1,7 @@
+class <%= class_name %> < ActionView::Component::Base
```

Comment:
I'm not sure if this should be `ApplicationComponent` instead of `ActionView::Component::Base`

Reply:
I would only inherit from `ApplicationComponent` if it starts shipping with Rails (like `ApplicationRecord` or it the generator creates that file as well. Do you think we should generate that file too?

Reply:
This is a tricky one. Could we look for an `ApplicationComponent` and inherit from it if it exists, otherwise inheriting from `ActionView::Component::Base`?